packages/now-cli/src/util/dev/builder.ts

```diff
@@ -364,6 +368,8 @@ export async function executeBuild(
       `Built ${match.use}:${entrypoint} [${ms(endTime - startTime)}]`
     );
   }
+
+  console.log(JSON.stringify({ result }, null, 2));
```

Comment:
I do this all the time when debugging builders… Maybe we should include a "dev for `now dev`" debug logging mode for us.

Reply:
Maybe even a debug level or scope, since most of the time I want specific information about something, and not everything